<launch>
    <node name="provider_sonar"
        pkg="provider_sonar"
        type="provider_sonar_node"
        output="log" />

    <node name="Sonar_to_world"
        pkg="tf"
        type="static_transform_publisher" 
        args="0.5 0.0 0.0 1.57 1.57 1.57 /sparus2 /Micron  100" />

    <rosparam command="load"
        file="$(find provider_sonar)/config/sonar_configuration.yaml"/>
</launch>
